src/main/java/com/assignment/students/controller/WebController.java

```diff
@@ -1,85 +1,112 @@
 package com.assignment.students.controller;
 
-import com.assignment.students.model.Class;
+import com.assignment.students.model.Course;
 import com.assignment.students.model.Student;
-import com.assignment.students.service.ClassService;
+import com.assignment.students.service.CourseService;
+import com.assignment.students.service.CourseStudentService;
 import com.assignment.students.service.StudentService;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.validation.BindingResult;
 import org.springframework.web.bind.annotation.*;
 
 import javax.validation.Valid;
+import java.util.ArrayList;
+import java.util.List;
 
 @Controller
 public class WebController {
 
-    private final ClassService classService;
+    private final CourseService courseService;
     private final StudentService studentService;
+    private final CourseStudentService courseStudentService;
 
-    public WebController(ClassService classService, StudentService studentService) {
-        this.classService = classService;
+    public WebController(CourseService courseService, StudentService studentService, CourseStudentService courseStudentService) {
+        this.courseService = courseService;
         this.studentService = studentService;
+        this.courseStudentService = courseStudentService;
     }
 
     @GetMapping("/v1/students")
     public String studentForm(Model model){
-        model.addAttribute("students",new Student());
-        return"students";
+        model.addAttribute("student",new Student());
+        return"student-form";
     }
 
-    @GetMapping("/v1/classes")
+    @GetMapping("/v1/courses")
     public String courseForm(Model model){
-        model.addAttribute("class",new Class());
-        model.addAttribute("classes", classService.getAllClasses());
+        model.addAttribute("course",new Course());
+        model.addAttribute("courses", courseService.getAllCourses());
         return "course-form";
     }
 
 
-    @GetMapping("/v1/classes/{id}/delete")
+    @GetMapping("/v1/courses/{id}/delete")
```

Comment:
I think it would be better to use POST instead of GET for delete. GET normally means nothing is changing.

Reply:
Updated it use POST. Had to change hyperlinks to buttons to enable this